src/parse.ts

```diff
@@ -1118,6 +1129,17 @@ function CHECKTAGCATEGORY(category, params) {
     // category === 'Basic' or 'MediaorMasterPlaylist' or 'Unknown'
 }
 
+function parseCustomTag(params: string): CustomTags {
+    const customTags = {};
+    if (params) {
+        for (const pair of params.split(';')) {
```

Comment:
Is there any sort of escape character we need to worry about? Like `#EXT-X-CUSTOM-TAGS:math=a\=b`